src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -8,7 +8,7 @@ azure-cli-core==2.30.0
 azure-cli-telemetry==1.0.6
 azure-cli==2.30.0
 azure-common==1.1.22
-azure-core==1.19.0
+azure-core==1.20.1
```

Comment:
May I know why do you need to upgrade `azure-core`?

Reply:
azure-synapse-artifacts==0.10.0 needs azure-core upgrade^_^

Reply:
want to confirm with @jiasli whether upgrading `azure-core` will impact other modules or not

Reply:
I think it is fine if CI passes.

Reply:
What does CI mean?

Reply:
ADO pipeline triggered by each PR

Reply:
It seems upgrading `azure-core` does impact other modules. After this PR get merged, our CI crashes...

65 tests fail because of `azure.core.pipeline.transport._requests_basic.StreamDownloadGenerator` throwing `azure.core.exceptions.IncompleteReadError: ('Connection broken:......)`. And 62 of them are webapp tests.
See [pipeline](https://dev.azure.com/azure-sdk/public/_build/results?buildId=1216694&view=logs&jobId=771f6cdf-8d98-5d9b-2fb9-d3a6e9554788&j=771f6cdf-8d98-5d9b-2fb9-d3a6e9554788&t=0a182f1d-9946-580d-467e-4d4ebd6af32d):
![image](https://user-images.githubusercontent.com/10227683/143820765-c69a8218-ba3f-484b-a192-383e1552ce65.png)

@jiasli @zhoxing-ms Can you help take a look?
@zesluo Do you mind if I revert this PR if other modules' failure can't be fixed before the code complete date(12/01)?